_docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhdp2.theme

```diff
@@ -21,6 +23,39 @@ function rhdp2_preprocess_page(&$variables) {
   $variables['main_menu'] = _rhdp2_build_menu('rhd-navigation', ['pf-c-nav__tertiary-list']);
   $variables['topics_menu'] = _rhdp2_build_menu('topics-menu', ['pf-c-nav__tertiary-list']);
   $variables['mobile_menu'] = _rhdp2_build_menu('rhd-navigation', ['pf-c-nav__tertiary-list'], 'menu__rhd_navigation__mobile');
+  $variables['akamai_cache_clear_button'] = _rhdp2_build_block('akamai_cache_clear_block');
+}
+
+ /**
```

Comment:
There are several PHPCs warnings for this block
- Line indented incorrectly; expected 0 spaces, found 1
- Parameter tags must be grouped together in a doc comment
- Type hint "array" missing for $config

Can you please clear the phpcs warnings here and the ones introduced by the ics changes?

Reply:
Resolved in commit d72870a